kyo-core/shared/src/main/scala/kyo/package.scala

```diff
@@ -91,6 +91,6 @@ package object kyo:
             if cond then bug(msg)
 
         def apply(msg: String): Nothing =
-            throw KyoBugException(msg + " Please file a ticket.")
+            throw KyoBugException(msg + " Please open an issue 🥹 https://github.com/getkyo/kyo/issues")
```

Comment:
could probably remove the https:// to make it a bit shorter (hopefully fitting on people's terminals)...

Reply:
I think the full URL is fine, I imagine some editors could even make it clickable that way. I used 🙈 for the dramatic effect in the [scheduler code](https://github.com/getkyo/kyo/blob/c4a8a955c157bb2a36b731d6f1c4907db1d84383/kyo-scheduler/jvm/src/main/scala/kyo/scheduler/package.scala#L16) 😅

Reply:
> some editors could even make it clickable that way
Yeah, that's what I was imagining. Make it one command-click away :)